src/main/kotlin/org/jitsi/videobridge/health/config/HealthConfigk.kt

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright @ 2018 - present 8x8, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jitsi.videobridge.health.config
+
+import org.jitsi.utils.configk.dsl.multiProperty
+import org.jitsi.videobridge.config.JvbConfigk
+import org.jitsi.videobridge.config.legacyProperty
+import org.jitsi.videobridge.config.newProperty
+import org.jitsi.videobridge.config.simple
+import java.time.Duration
+
+class HealthConfigk {
```

Comment:
in the real impl, this would replace `HealthConfig` and would be named that (no trailing `k`)